src/main/resources/json-schema/identifications.json

```diff
@@ -117,6 +117,13 @@
               "Roptrocerus typographi (Györfi, 1952)"
             ]
           },
+          "ods:scientificNameHtmlLabel": {
```

Comment:
Don't you need to add a Term for this? Why isn't this in the services? `.withDwcScientificNameHtmlLabel()`

Reply:
Yeah we fill this in only at the name resolution service. However, I did need to add a field for this in the digitalSpecimen. It won't be in any data we get delivered and is optional. I did want the schema's to be aligned so I update this one already in the PR.